.travis.yml

```diff
@@ -7,8 +7,14 @@ python:
     - "3.6"
 
 install:
-    - wget https://raw.githubusercontent.com/phanrahan/magma/master/.travis/install_coreir.sh
```

Comment:
can you just note as a comment in this script why this is happening? in case it doesn't get fixed soon 🙂

Reply:
Done

Reply:
Now we don't need this right?